openslides_backend/action/actions/meeting/unarchive.py

```diff
@@ -23,13 +23,15 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         meeting = self.datastore.get(
             fqid_from_collection_and_id(self.model.collection, instance["id"]),
             ["committee_id", "is_active_in_organization_id"],
+            lock_result=False,
```

Comment:
remove, since this relation gets changed in this action

Reply:
Done.